stable/sematext-agent/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+appVersion: "1.0"
+version: 1.0.0
+description: Helm chart for deploying Sematext Agent to Kubernetes
+keywords:
+  - sematext
+  - observability
+  - monitoring
+  - logging
+  - containers
+  - ebpf
+home: https://sematext.com/cloud
```

Comment:
Any interest in adding the icon field so that it is displayed correctly in https://hub.helm.sh/?

Reply:
Yup, added.